calico_node/tests/manifests/crds.yaml

```diff
@@ -0,0 +1,69 @@
+kind: List
```

Comment:
Do we need this file? Don't we already have this checked in somewhere?

Reply:
It looks like we have one in the upgrade section, I'll have it point to that one instead.

Reply:
Don't think we have one with only CRDs in this repo, but we can apply the one in libcalico-go by using it's url, so if the CRDs change in future we don't have to remember to update them here

Reply:
The one in upgrade is split into 2 because name overlap between crd and tpr names

Reply:
We can also do what Felix does and pull from the vendor dir: https://github.com/projectcalico/felix/blob/master/k8sfv/run-test#L63

Reply:
Pulling from the vendor dir seems like the most straightforward if we're having success with it in Felix, that's the one I grabbed anyway.